apps/showcase/src/assets/trainings/sdk/steps/model-extension/instructions.md

```diff
@@ -0,0 +1,41 @@
+There are certain cases in which you want to be able to extend your SDK models, and therefore ensure that revivers are generated.
+
+Let's continue with the use case of the previous exercise.\
+In order to keep track of the user's current booking, it would be useful to generate an ID.
+To do this, we can create a new model which extends the previously generated `Flight` type.
+
+> [!NOTE]
+> For this exercise, we provide an SDK that has been generated with model extension.
+> 
+> To verify this, check out the configuration used to generate the API (either the command line or the file `openapitools.json`).\
+> Ensure that the global property `allowModelExtension` has been set to `true`.\
+> This will guarantee that the revivers of the base models are generated, which is essential for the following exercise.
+
+### Exercise
+
+Extensions of base models (located in the `base` folder) are created in the `core` folder.
+
+#### Creating the extended model
+First, let's create the type `FlightCoreIfy` in `libs/sdk/src/models/core/flight.ts`.
+This type should extend the type `Flight`, imported from the `base` folder and add a new field `id: string`.
+
+> [!WARNING]
+> The naming convention requires the core model to contain the suffix `CoreIfy`.\
+> You can find more information on core models in the
+> <a href="https://github.com/AmadeusITGroup/otter/blob/main/docs/api-sdk/SDK_MODELS_HIERARCHY.md" target="_blank">SDK models hierarchy documentation</a>.
+
+#### Creating the extended reviver
+Then, you can create the reviver for this new core model in `libs/sdk/src/models/core/flight.reviver.ts`.\
+This extended reviver will call the reviver of the base `Flight` model and add the `id` to the returned object.\
+The base reviver should exist since we have ensured this during the prerequisites of the exercise.
```

Comment:
maybe move the `[!NOTE]` to the `### Exercise` section and add **Prerequisite** at the beginning of the note to be more clear?

Reply:
I've created a dedicated section at the beginning of the exercise to explain the base model and reviver, so I've removed this sentence